aws/resource_aws_db_instance.go

```diff
@@ -404,6 +404,25 @@ func resourceAwsDbInstance() *schema.Resource {
 				},
 			},
 
+			"performance_insights_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
We should not set [`Computed`](https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#computed) on this attribute or any of the others. Operators should know if PI has been configured outside Terraform. See also: https://www.terraform.io/docs/extend/best-practices/detecting-drift.html

Reply:
Correct 😄